Add Custom Fields to JiraCreateIssueExample Script (#22601)
demisto/content: Packs/Jira/ReleaseNotes/2_1_0.md

##### Jira
- Added an argument, *customFields* to be able to set custom fields when creating a Jira issue.

#### Scripts
##### JiraCreateIssue-example

> - Added an argument, *customFields* to be able to set custom fields when creating a Jira issue.

Reviewer: ```suggestion
Added the *customFields*  argument to support custom fields when creating a Jira issue.
```
